packages/m365_defender/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.3.2"
```

```suggestion
- version: "3.4.0"
```
(enhancements bump minor)